### PR TITLE
Label the CQM submitted to the hybrid solver

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: dwave/plugins/sklearn/transformers.py [hunk 1]
```diff
@@ -293,7 +293,7 @@ def fit(
                 """
             )
 
-        sampleset = sampler.sample_cqm(cqm, time_limit=self.time_limit)
+        sampleset = sampler.sample_cqm(cqm, time_limit=self.time_limit, label="feature-selection")
```
Comment (JoelPasvolsky) [inline dwave/plugins/sklearn/transformers.py:296]:
```suggestion
        sampleset = sampler.sample_cqm(cqm, time_limit=self.time_limit, label="dwave-scikit-learn-plugin Feature Selection")
```

Our label convention doesn't include this type of repo (not dwave-example, JN, or demo), the closest we have is [VirtualGraph flux bias measurements](https://github.com/dwavesystems/dwave-system/blob/bedfe5143a8579348be07e4ef5e8fe0646ce81ff/dwave/system/flux_bias_offsets.py), but this seems more consistent, especially when we have multiple FS repos.

File: releasenotes/notes/problem-submission-label-68c3670bd04b5c25.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+features:
+  - Apply the label ``"feature-selection"`` to the problems submitted to Leap.
```
Comment (JoelPasvolsky) [inline releasenotes/notes/problem-submission-label-68c3670bd04b5c25.yaml:3]:
```suggestion
  - Apply the label ``"dwave-scikit-learn-plugin Feature Selection"`` to the problems submitted to Leap.
```

**Review by JoelPasvolsky — APPROVED** [review]
Feel free to reject the suggested changes.

**Comment by arcondello** [comment]
I am inclined to go with the shorter one. I think this is meant to be a hint more than actually informative.

**Comment by arcondello** [comment]
Ok, now the label will be `dwave.plugins.sklearn.transformers.SelectFromQuadraticModel`

image
